streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableReduceTest.java

```diff
@@ -19,8 +19,9 @@
 import org.apache.kafka.streams.processor.Processor;
 import org.apache.kafka.streams.processor.internals.AbstractProcessorContext;
 import org.apache.kafka.streams.processor.internals.ProcessorNode;
-import org.apache.kafka.streams.state.KeyValueStore;
-import org.apache.kafka.test.GenericInMemoryKeyValueStore;
+import org.apache.kafka.streams.state.TimestampedKeyValueStore;
+import org.apache.kafka.streams.state.ValueAndTimestamp;
+import org.apache.kafka.test.GenericInMemoryTimestampedKeyValueStore;
```

Comment:
Can we replace other uses of `GenericInMemoryKeyValueStore` so we can remove the old class as well?